README.md

```diff
@@ -104,51 +70,10 @@ Based on the attributes set on the template during its creation, ensure that all
 
 Note: This service is an internal service so it doesn't have the frontend.
 
-# Run with Gunicorn
-
-In deployed environments the service is run with gunicorn. You can run the service locally with gunicorn to test
-
-First set the FLASK_ENV environment you wish to test eg:
+## How to post data for notification service.
 
-    export FLASK_ENV=dev
-
-Then run gunicorn using the following command:
-
-    gunicorn wsgi:app -c run/gunicorn/local.py
-
-
-### Build with Paketo
-
-[Pack](https://buildpacks.io/docs/tools/pack/cli/pack_build/)
-
-[Paketo buildpacks](https://paketo.io/)
-
-```pack build <name your image> --builder paketobuildpacks/builder:base```
-
-Example:
-
-```
-[~/work/repos/funding-service-design-notification] pack build paketo-demofsd-app --builder paketobuildpacks/builder:base
-***
-Successfully built image paketo-demofsd-app
-```
-
-You can then use that image with docker to run a container
-
-```
-docker run -d -p 8080:8080 --env PORT=8080 --env FLASK_ENV=dev [envs] paketo-demofsd-app
-```
-
-`envs` needs to include values for each of:
-GOV_NOTIFY_API_KEY
-SENTRY_DSN
-GITHUB_SHA
-
-```
-docker ps -a
-CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS                    PORTS                    NAMES
-42633142c619   paketo-demofsd-app          "/cnb/process/web"       8 seconds ago    Up 7 seconds              0.0.0.0:8080->8080/tcp   peaceful_knuth
-```
+Go to relevant service. See example
```

Comment:
Can you replace this with actual links to each of the other readmes please

Reply:
paketo section with link is in this section now

Reply:
So the paketo bit is fine, but each type of notification has it's own readme - please link to those as they contain examples of how to use those notifications